src/main/scala/com/databricks/spark/xml/util/TypeCast.scala

```diff
@@ -110,6 +112,17 @@ object TypeCast {
     }
   }
 
+  private[xml] def isDate(value: String, dateFormatter: SimpleDateFormat = null) = {
```

Comment:
@jameswinegar, seems there looks no case where `dateFormatter` should be omitted. I think we can just `isDate(value: String, dateFormatter: SimpleDateFormat)`